pages/blog/status-update-47-20.md

```diff
@@ -0,0 +1,160 @@
+
+Yes, our official [GitHub Action for Generator](https://github.com/marketplace/actions/generator-for-asyncapi-documents) already uses the latest Generator and is released under v1. 
+
+```yaml
+- name: Generating HTML from my AsyncAPI document
+  uses: asyncapi/github-action-for-generator@v1
+  with:
+    template: '@asyncapi/html-template@0.15.4'  #In case of template from npm, because of @ it must be in quotes
+    filepath: docs/api/my-asyncapi.yml
+    parameters: baseHref=/test-experiment/ sidebarOrganization=byTags #space separated list of key/values
+    output: generated-html
+```
+
+[The AsyncAPI Playground](https://playground.asyncapi.io/) is up to date with the latest generator. 
+
+### Writing Own Generator Template Ain't Easy
+
+Writing a template compatible with the AsyncAPI Generator is not an easy task that you can complete in one day. To write a template that generates docs or code, you need to have not only decent knowledge about AsyncAPI specification but also the features of the Generator:
+
+- How do I extract data from the spec file?
+- What template engine powers Generator and how to use it?
+- How can I add optional features to the template?
+
+There is a lot of it. We wanted to make it easy for you. We created a [GitHub Template](https://github.com/asyncapi/template-for-generator-templates) that showcases all features available in the Generator. It contains:
+- A template that generates class diagram showing relations between schemas
+- Detailed readme which explains all the features and where are they used in the template
+
+To try out the template, run :point_down: command:
+```
+# Install the AsyncAPI Generator if you do not have it yet
+npm install -g @asyncapi/generator
+
+# Run generation
+ag https://raw.githubusercontent.com/asyncapi/generator/v1.0.1/test/docs/dummy.yml https://github.com/asyncapi/template-for-generator-templates -o output
+
+# Open the result of the generation
+open output/index.html
+```
+
+<Figure
+  src="/img/posts/status-update-47/diagram.webp"
+  caption="Figure 1: Schema consumed by the template and the resulting diagram"
+/>
+
+## React Component and Web Component
+
+### New Npm Scope
+
+Since the 0.16.2 release, we changed the npm scope of the component and now use the official **asyncapi** one. That means you should update your dependencies and from now on use **@asyncapi/react-component**.
+
+[![Edit asyncapi-react-component-in-action](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/asyncapi-react-component-in-action-wvdy2)
+
+### Web Component
+
+Since the 0.17.5 release, we are now automatically publishing also the @asyncapi/web-component to npm. Since it depends on the React component, it will always follow the same version number. This is all possible thanks to work done by amazing [Hesyar Uzuner](https://github.com/hesyar), [Claude Gex](https://github.com/gexclaude), [Maciej Urbańczyk](https://github.com/magicmatatjahu).
```

Comment:
Double `since`.

```suggestion
Since the 0.17.5 release, we are now automatically publishing also the @asyncapi/web-component to npm. Given that it depends on the React component, it will always follow the same version number. This is all possible thanks to work done by amazing [Hesyar Uzuner](https://github.com/hesyar), [Claude Gex](https://github.com/gexclaude), [Maciej Urbańczyk](https://github.com/magicmatatjahu).
```